package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "recreated-from-lock",
+  "version": "1.0.0",
+  "dependencies": {},
+  "devDependencies": {
+    "elm-verify-examples": "5.2.0"
+  }
+}
```

Comment:
off topic but I realise this file is missing. reconstructed with https://gist.github.com/choonkeat/5a7a6edd7137c3aeac2c3023aa60b90a